taskqueue/taskqueue.go

```diff
@@ -104,7 +117,15 @@ func (tq *WorkerTaskQueue) worker(executor Executor) {
 			}
 		}
 		for _, task := range tasks {
+			tq.activeTasks++
```

Comment:
is this atomic(-enough) to safely do this for a shared `int` field in a struct between go routines, or do I need to put a mutex around these?

Reply:
no, apparently not, "DATA RACE", am trying out `sync/atomic` for this